docs/sdk/example-apps.md

```diff
@@ -17,7 +17,7 @@ Welcome to the Samples & Example Apps page, which showcases example applications
 | [Flutter](https://github.com/devcyclehq/flutter-client-sdk)               | [Flutter Reference Docs](/sdk/client-side-sdks/flutter.md)                     |
 | [Go](https://github.com/DevCycleHQ/go-server-sdk)                         | [Go Reference Docs](/sdk/server-side-sdks/go.md)                               |
 | [iOS](https://github.com/devcyclehq/ios-client-sdk)                       | [iOS Reference Docs](/sdk/client-side-sdks/ios.md)                             |
-| [Java (Cloud Bucketing)](https://github.com/DevCycleHQ/java-server-sdk)   | [Java (Cloud Bucketing) Reference Docs](/sdk/server-side-sdks/java-cloud.md)   |
+| [Java (Cloud Bucketing)](https://github.com/DevCycleHQ/java-server-sdk)   | [Java (Cloud Bucketing) Reference Docs](/sdk/server-side-sdks/java-cloud)   |
```

Comment:
Does this actually work? I thought it needed to be changed to `/java-cloud/java-cloud.md`

Reply:
it works! it links to the same page

Reply:
although maybe i should keep it consistent with the other links here